scwx-qt/source/scwx/qt/model/marker_model.cpp

```diff
@@ -254,6 +254,10 @@ bool MarkerModel::setData(const QModelIndex& index,
 
 void MarkerModel::HandleMarkersInitialized(size_t count)
 {
+   if (count == 0)
+   {
+      return;
+   }
```

Comment:
Is the same protection needed in HandleMarkerAdded? While marker_count should never be zero when a marker is added, might it be possible for a marker to be added and removed prior to the slot being called?